### PR TITLE
br stream: set default startTS and print steam task into screen

### PR DESCRIPTION
**Pull request by joccau**
### What problem does this PR solve?

Issue Number: close https://github.com/pingcap/tidb/issues/29970

### What is changed and how it works?

- Set currentTS to startTS as a default value
- Print stream task info into screen
- Limit single task currently，but we make a implement of multi-task on code

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [x] Manual test (add detailed scripts or steps below)
- [ ] No code

### Release note

```release-note
None
```

### CODE REVIEW

File: br/pkg/stream/models.go [hunk 1]
```diff
@@ -29,6 +29,12 @@ var (
 	taskNameRe = regexp.MustCompile(`^[0-9a-zA-Z_]+$`)
 )
 
+// PrefixOfTask is the prefix of all task
+// It would be `<prefix>/info/`
+func PrefixOfTask() string {
+	return path.Join(streamKeyPrefix, taskInfoPath) + "/"
```
Comment (kennytm) [inline br/pkg/stream/models.go:35]: change `TaskOf` to use this?

File: br/pkg/task/stream.go [hunk 2]
```diff
@@ -31,18 +31,21 @@ import (
...
+	"go.uber.org/zap/zapcore"
 )
 
 const (
-	flagStreamTaskName = "taskName"
-	flagStreamStartTS  = "startTS"
-	flagStreamEndTS    = "endTS"
-	flagGCSafePointTTS = "gcTTL"
+	flagStreamTaskName        = "task-name"
+	flagStreamTaskNameDefault = "ALL"
```
Comment (3pointer) [inline br/pkg/task/stream.go:45]:
```suggestion
	flagStreamTaskNameDefault = "all"
```

File: br/pkg/task/stream.go [hunk 3]
```diff
@@ -200,23 +205,27 @@ func (s *streamMgr) setLock(ctx context.Context) error {
 	return s.bc.SetLockFile(ctx)
 }
 
-// checkStartTS checks that startTS should be smaller than currentTS,
+// adjustAndCheckStartTS checks that startTS should be smaller than currentTS,
 // and endTS is larger than currentTS.
-func (s *streamMgr) checkStartTS(ctx context.Context) error {
+func (s *streamMgr) adjustAndCheckStartTS(ctx context.Context) error {
```
Comment (3pointer) [inline br/pkg/task/stream.go:210]: please add a unit test

File: errors.toml [hunk 4]
```diff
@@ -241,6 +241,11 @@ error = '''
 the system table isn't supported for restoring yet
 '''
 
+["BR:Stream:ErrStreamLogTaskExist"]
+error = '''
+it has stream task existed
```
Comment (3pointer) [inline errors.toml:246]: stream task already exists

**Review by 3pointer — APPROVED** [review]

**Review by YuJuncen — APPROVED** [review]

**Pull request merged**